Tests/Tests/REventLogger/REventsStorageSpec.swift

```diff
@@ -0,0 +1,81 @@
+import Quick
+import Nimble
+import Foundation
+
+#if canImport(RSDKUtils)
+@testable import RSDKUtils // Cocoapods version
+#else
+@testable import RSDKUtilsMain
+@testable import REventLogger
+#endif
+
+class REventsStorageSpec: QuickSpec {
+    override func spec() {
+        let userDefaults = UserDefaults(suiteName: "REventsStorageSpec")!
+
+        afterEach {
+            UserDefaults.standard.removePersistentDomain(forName: "REventsStorageSpec")
+        }
+
+        describe("REvents Logger Storage ") {
+            context("getAllEvents method") {
+                it("will return empty array if no cached data was found") {
+                    let eventCache = EventDataCache(userDefaults: userDefaults)
+                    expect(eventCache.getAllEvents()).to(equal([]))
+                    expect(eventCache.getEventCount()).to(equal(0))
+                }
+                it("will return array of Events if cached data is avaialble") {
+                    let eventCache = EventDataCache(userDefaults: userDefaults)
+                    let eventId = EVentLoggerMockData.REventModel.getEventIdentifier()
```

Comment:
typo: `EVentLoggerMockData`
I would suggest to rename it like `REventLoggerMockData`